mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
```

Comment:
Error handling is not properly done in this code snippet. If the order is not found, it returns a 404 status with a message "Order does not exist", but if the order is found, it still returns a 200 status with a message "Order deleted successfully". This is inconsistent and can be misleading. It should return a 200 status with a message "Order deleted successfully" only if the order is found and deleted successfully. If the order is not found, it should return a 404 status with a message "Order not found".